src/Mod/Part/App/TopoShape.h

```diff
@@ -727,22 +736,175 @@ class PartExport TopoShape: public Data::ComplexGeoData
                                        const Mapper &mapper,
                                        const std::vector<TopoShape> &sources,
                                        const char *op=nullptr);
+    /**
+     * When given a single shape to create a compound, two results are possible: either to simply
+     * return the shape as given, or to force it to be placed in a Compound.
+     */
+    enum class SingleShapeCompoundCreationPolicy {
+        RETURN_SHAPE,
+        FORCE_COMPOUND
+    };

```

Comment:
Up around line 101 we have an enum class that uses lowercaseCapword rather than UPPER_CASE.  We should probably make a call and be consistent.

Reply:
Please use camel case rather than upper case -- I've been guilty of screwing this up myself so I apologize for the bad example. Modern C++ basically eschews all uppercase because it looks like macros.

Reply:
Got it!

Changed all enum classes members to lowercaseCapword, you should find this modification in the next commit